apps/admin_panel/assets/src/omg-account-select/index.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Avatar } from '../omg-uikit'
+
+const StyledAvatar = styled(Avatar)`
+  display: inline-block;
+  vertical-align: middle;
+  margin-right: 10px;
+`
+const AccountSelectContainer = styled.div`
+  position: relative;
+  white-space: nowrap;
+`
+const DetailContainer = styled.div`
+  display: inline-block;
+  vertical-align: middle;
+`
+const Name = styled.div``
+const Identifier = styled.div`
+  color: ${props => props.theme.colors.B100};
+  font-size: 10px;
+`
+const AccountSelect = ({ account }) => {
+	const name = _.get(account, 'name')
+	const identifier = _.get(account, 'id')
+	const thumbnail = _.get(account, 'avatar.thumb');
+
+	return (
+		<AccountSelectContainer>
```

Comment:
@jarindr  🤕 whyyyyy is the spacing like this... my config on vscode is 2 spaces

Reply:
which formatter you’re using tho

Reply:
@jarindr actually i'm not using any formatter 🤔 
maybe because when i created the file it defaulted to 4 spaces, then i changed it, so github still shows it this way? can you see in your code editor if its correctly showing 2 spaces... maybe its just github?